README.md

```diff
@@ -24,32 +27,56 @@ and also requires some notion of ***Location*** as a dependency for ***Insertion
-+ ***ElectrodeConfig.Electrode*** - corresponding electrodes in ***ProbeType.Electrode*** that are used for recording in this electrode configuration 
-(e.g. for Neuropixels 1.0 or 2.0, there can be at most 384 electrodes usable for recording per probe)
++ ***ElectrodeConfig.Electrode*** - corresponding electrodes in ***ProbeType.Electrode*** that are used for recording in this electrode configuration (e.g. for Neuropixels 1.0 or 2.0, there can be at most 384 electrodes usable for recording per probe)
 
 ### Extracellular ephys recording
 
-+ ***ProbeInsertion*** - a surgical insertion of a probe onto the animal. 
-Every experimental session consists of one or more ***ProbeInsertion***, with corresponding ***InsertionLocation***
-+ ***EphysRecording*** - each ***ProbeInsertion*** is accompanied by a corresponding ***EphysRecording***, 
-specifying the ***ElectrodeConfig*** used for the recording from the ***Probe*** defined in such ***ProbeInsertion***
-    + ***get_npx_data_dir*** method - the class ***EphysRecording*** requires user 
-    to supply a method to retrieve the directory containing the recorded neuropixels data (e.g. `*.ap.meta`, `*.ap.bin`, etc.), 
-    where the method's input arguments are the primary attributes identifying one ***EphysRecording***
++ ***ProbeInsertion*** - a surgical insertion of a probe in the brain. Every experimental session consists of one or more entries in ***ProbeInsertion*** with a corresponding ***InsertionLocation*** each
++ ***EphysRecording*** - each ***ProbeInsertion*** is accompanied by a corresponding ***EphysRecording***, specifying the ***ElectrodeConfig*** used for the recording from the ***Probe*** defined in such ***ProbeInsertion***
     
 ### Clusters and spikes
 
 This ephys element features automatic ingestion for spike sorting results from the ***kilosort*** method. 
 
 + ***Clustering*** - specify instance(s) of clustering on an ***EphysRecording***, by some ***ClusteringMethod***
-    + ***get_ks_data_dir*** method - the class ***Clustering*** requires user 
-    to supply a method to retrieve the directory containing the kilosort results, 
-    where the method's input arguments are the primary attributes identifying one ***Clustering***
-+ ***Unit*** - Identified unit(s) from one ***Clustering***, with associated ***ClusterQualityLabel***
-    + ***UnitSpikeTimes*** - spike times per unit
-    + ***Waveform*** - mean waveform across spikes per unit per recording electrode
++ ***Curation*** - specify instance(s) of curations performed on the output of a given ***Clustering***
++ ***CuratedClustering*** - set of results from a particular round of clustering/curation
+    + ***CuratedClustering.Unit*** - Identified unit(s) from one ***Curation***, and the associated properties (e.g. cluster quality, spike times, spike depths, etc.)
+    + ***WaveformSet*** - A set of spike waveforms for units from a given CuratedClustering
+
```

Comment:
Do we need explanation of part tables here too?

Reply:
Yes, agreed, put perhaps on the `main` branch, so that the same changes can be reflected here as well (maybe in the next PRs)